app/views/layouts/application.html.erb

```diff
@@ -33,22 +33,19 @@
         </li>
       </ul>
       <ul class="nav app-header__user-nav-container">
-        <% if @login_user %>
-
-        <li class="nav-item app-header__nav_item">
-          <%= link_to "Logged in as #{@login_user.username}", user_path(@login_user), class: "btn btn-primary" %>
-        </li>
-        <li class="nav-item app-header__nav_item">
-          <%= link_to "Log Out", logout_path, method: :post, class: "btn btn-primary" %>
-        </li>
-
+        <% if session[:user_id] %>
+          <% @user = User.find_by(id: session[:user_id])%>
+          <li class="nav-item app-header__nav_item">
+            <%= link_to "Logged in as #{@user.name}", user_path(@user.id), class: "btn btn-primary" %>
```

Comment:
for some reason this @user.name is not appearing in my button - it just says "Logged in as"- probably because I dont' have a github "name"